features/app_and_device_attributes.feature

```diff
@@ -5,67 +5,67 @@ Feature: App and Device attributes present
 
   Scenario: App and Device info is as expected
     When I run "AppAndDeviceAttributesScenario"
-    And I wait to receive a request
-    Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
-    And the "Bugsnag-API-Key" header equals "12312312312312312312312312312312"
+    And I wait to receive an error
+    Then the error is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
```

Comment:
Do we still need to have this explicit checks?

It seems that maze runner could do this automatically, now that it know what kind of payload to expect.

Reply:
Yes, agreed - and we already have PLAT-5782 to look at abstracting them away.